code/controllers/mc/master.dm

```diff
@@ -124,15 +124,21 @@ var/CURRENT_TICKLIMIT = TICK_LIMIT_RUNNING
 // 	Make a subsystem, give it the SS_NO_FIRE flag, and do your work in it's Initialize()
 /datum/controller/master/proc/Setup()
 	set waitfor = FALSE
-	sleep(1 SECONDS)
-	to_chat(world, "<span class='boldannounce'>Initializing subsystems...</span>")
-
+	sleep(-1)
```

Comment:
Why did you replace this?

Reply:
sleep(-1) works much better in this case

Reply:
How so?